### ISSUE TITLE
Loinc 2965-2:  "Specific gravity of Urine"

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#3512]
Suggest creating a new term for Loinc: 2965-2 [Specific gravity of Urine] 
New term label:
New term comment (if any):
Your biocurator ID for loinc2hpo (if desired): HPO:probinson

Low/high urine specific gravity terms

see https://www.ncbi.nlm.nih.gov/pubmed/25280126

**Comment by pnrobinson** [comment]
See this: https://hpo.jax.org/app/browse/term/HP:0003158
However, I am not sure if the LOINC term really measures "abnormally' low urine specific gravity in a wat that would justify the diagnosis of Hyposthenuria

**Comment by pnrobinson** [comment]
I do not think we can map this LOINC term, because I think its interpretation is very context dependent.
